rotkehlchen/externalapis/etherscan.py

```diff
@@ -548,6 +548,9 @@ def has_activity(self, account: ChecksumEvmAddress) -> EtherscanHasChainActivity
         result = self._query(module='account', action='tokentx', options=options)
         if len(result) != 0:
             return EtherscanHasChainActivity.TOKENS
+        balance = self._query(module='account', action='balance', options={'address': str(account)})  # noqa: E501
+        if int(balance) != 0:
+            return EtherscanHasChainActivity.TRANSACTIONS
```

Comment:
seems that we don't make a distinction and we only check for tokens but I would add a `BALANCE` entry to the enum and use it here

Reply:
Also update the docstring of this function regarding what it does